house_of_force.c

```diff
@@ -37,25 +37,25 @@ int main(int argc , char* argv[])
 
 	fprintf(stderr, "\nNow the heap is composed of two chunks: the one we allocated and the top chunk/wilderness.\n");
 	int real_size = malloc_usable_size(p1);
-	fprintf(stderr, "Real size (aligned and all that jazz) of our allocated chunk is %d.\n", real_size);
+	fprintf(stderr, "Real size (aligned and all that jazz) of our allocated chunk is %d.\n", real_size + sizeof(long)*2);
```

Comment:
I guess you want to include the header into the size here?
In this case it makes more sense to print the actual chunk size without the header imo.